src/apps/vpn/platforms/ios/ioslogger.swift

```diff
@@ -4,49 +4,110 @@
 
 import Foundation
 import os.log
+import OSLog
 
-public class Logger {
-    static var global: Logger?
-
-    var tag: String
+public class IOSLoggerImpl : NSObject {
+    private let log: OSLog
+    private let dateFormatter = DateFormatter()
```

Comment:
Nit: If you want to move the formatter setup out of the function, you could get fancy:
```suggestion
    private lazy var dateFormatter: DateFormatter = {
        let formatter = DateFormatter()
        formatter.dateFormat = "dd.MM.yyyy HH:mm:ss"   
        return formatter
    }() = DateFormatter()
```

The snippet of setup code will be run the first time the `dateFormatter` is called, and never again. Unfortunately, as of now it needs to be done with a `var` and not `let`.